test/Vesting.t.sol

```diff
@@ -369,4 +386,17 @@ contract BlueberryStakingTest is Test {
         console2.log("Together, all users received: %s", totalBLB);
         assertEq(totalBLB, totalVestAmount);
     }
+
+    function testGetPrice() public {
+        // Period 1: 0.02e6 BLB/USDC
```

Comment:
I've seen some disagreement with this nomenclature, so take this with a grain of salt, but I would write the units as `USDC/BLB` rather than `BLB/USDC`. This is 0.02e6 `USDC` _per_ `BLB` token. For whatever reason, though, it seems popular to write this the other way around as you've done here.

Reply:
Agreed that this is technically incorrect unit nomenclature. Will revise.